kcp/ckcp/setup-ckcp-on-openshift.sh

```diff
@@ -192,15 +192,14 @@ test-gitops-service-e2e-in-kcp() {
 
 test-gitops-service-e2e-in-kcp-in-ci() {
   
-  export KUBECONFIG=${TMP_DIR}/ckcp-ckcp.default.managed-gitops-compute.kubeconfig
   printf "The Kubeconfig being used for this is:" $KUBECONFIG
```

Comment:
We are printing the KUBECONFIG path here before setting its value. Should it go after line https://github.com/redhat-appstudio/managed-gitops/blob/661336ce4fdda9485af815092124f243e4a0b2b2/kcp/ckcp/setup-ckcp-on-openshift.sh#L197

Reply:
Yes, we should found a better way to deal with this coming right up in another commit o/